docker/start_server.sh

```diff
@@ -2,4 +2,4 @@
 
 # launch the server
 echo "launch icubam server in mode ${ENV_MODE}"
-python3 scripts/run_server.py --port 8888 --mode="${ENV_MODE}" --config=/home/icubam/resources/config.toml
+python3 scripts/run_server.py --port 8888 --config="/home/icubam/resources/config_${ENV_MODE}.toml"
```

Comment:
So here I would just consider there to only be a single config.toml, no need to even use the ENV_MODE variable.  I know that for myself I just run everything with prod all the time, and depending on where I am I change the config files, or point to a different config file if needed.